### PR TITLE
instruction of ssl certificate verify failed

### PR DESCRIPTION
**Pull request by oceanchad**
## Pull request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [x] Documentation content changes
- [ ] Other (please describe):

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?

- Instruction of ssl verification issue when install Poetry.

### CODE REVIEW

File: docs_dev/docs/dev/beginners.md [hunk 1]
```diff
@@ -46,6 +46,8 @@ Please install [Poetry](https://python-poetry.org/) for dependency management an
 
     curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python3 -
 
+If you got error like `'urllib.error.URLError: <urlopen error [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed'`, go check your python3 folder `/Applications/Python 3.10/` and click `Install Certificates.command` file.
```
Comment (pan93412) [inline docs_dev/docs/dev/beginners.md:49]: 簡單修了一下文法和語句：

```
When you encountered such an error like `'urllib.error.URLError: <urlopen error [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed'` on macOS, you can navigate to your Python 3 installation folder (such as `/Application/Python 3.10`) and run `Install Certificates.command`.
```

如果覺得不太好的話，也可以把它改得更好  ><

**Review by pan93412 — CHANGES_REQUESTED** [review]

**Review by pan93412 — APPROVED** [review]

**Comment by toomore** [comment]
Hi @oceanchad , we need signoff[1] the commit by following the DCO declaration. How to fix the problem please read the alert result here[2], thank you.

[1] https://volunteer.coscup.org/docs/dev/contribute-on-github/
[2] https://github.com/COSCUP/COSCUP-Volunteer/pull/89/checks?check_run_id=7095891856

**Comment by pan93412** [comment]
Let me help him to sign off correctly.

**Comment by pan93412** [comment]
cc @toomore

**Pull request closed without merging**